gateway/api/access_policies/jobs.py

```diff
@@ -0,0 +1,67 @@
+"""
+Access policies implementation for Job access
+"""
+import logging
+from django.contrib.auth import get_user_model
+from api.models import Job
+
+User = get_user_model()
+
+
+logger = logging.getLogger("gateway")
+
+
+class JobAccessPolocies:  # pylint: disable=too-few-public-methods
+    """
+    The main objective of this class is to manage the access for the user
+    to the Job entities.
+    """
+
+    @staticmethod
+    def can_access(user: User, job: Job) -> bool:
+        """
+        Checks if the user has access to save the result of a Job:
```

Comment:
Correct me if I'm wrong here, @paaragon . This method is to check if the user can retrieve the job so we should update the text to something like:

```
Checks if the user has access to the Job
```

Or something like that. It's not specific to `save` the result.